docs/Installation.md

```diff
@@ -223,13 +223,13 @@ The command is expected to give an immediate JSON response similiar to :
 Install dependencies available from binary packages:
 
 ```sh
-sudo apt-get install libclass-method-modifiers-perl libconfig-inifiles-perl libdaemon-control-perl libdata-dump-perl libdbd-sqlite3-perl libdbi-perl libfile-sharedir-perl libfile-slurp-perl libhtml-parser-perl libintl-perl libio-captureoutput-perl libjson-pp-perl libjson-rpc-perl libmodule-build-tiny-perl libmoose-perl libplack-perl librouter-simple-perl libstring-shellquote-perl libtest-requires-perl libtest-warn-perl libtext-microtemplate-perl libtie-simple-perl starman
+sudo apt-get install libclass-method-modifiers-perl libconfig-inifiles-perl libdaemon-control-perl libdata-dump-perl libdbd-sqlite3-perl libdbi-perl libfile-sharedir-perl libfile-slurp-perl libhtml-parser-perl libintl-perl libio-captureoutput-perl libjson-pp-perl libjson-rpc-perl libmoose-perl libplack-perl librouter-simple-perl libstring-shellquote-perl libtest-requires-perl libtest-warn-perl libtext-microtemplate-perl libtie-simple-perl starman
 ```
 
 Install dependencies not available from binary packages:
 
 ```sh
-sudo cpan -i Plack::Middleware::Debug Parallel::ForkManager Net::IP::XS
+sudo cpan -i Module::Build::Tiny JSON::RPC::Dispatch Plack::Middleware::Debug Parallel::ForkManager Net::IP::XS

```

Comment:
* Is JSON::RPC::Dispatch not included in libjson-rpc-perl?
* Do we even use Module::Build::Tiny?

Reply:
* No.
* It is required by `Plack::Middleware::Debug`. We do not need to explicitly state it since it is installed anyway as a dependency to `Plack::Middleware::Debug`.

Reply:
I suggest that libjson-rpc-perl is removed from the Debian installation, that JSON::RPC be installed from CPAN instead, possibly with a note about Debian 7 with libjson-rpc-perl-0.96 being the culprit.

I thought the idea was to not list transitive dependencies in the installation instruction. Am I mistaken?

Reply:
Fine.

Reply:
Our laundry list is getting long. I made a bunch of changes for the installation instruction already. Will you take care of this?